torchaudio/functional_filtering.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import math
+import torch
+import math
+import torchaudio
+
+"""
+Biquad filtering approach based on SoX implementation 14.4.2
+
+Performance is currently very slow
+"""
+
+__all__ = [
```

Comment:
I'm not for or against, but why did you decide to move those functionals outside of functionals? Are you suggesting all of sox re-implementation should be in separate files?

Reply:
Yes - may be nice to separate into functional layer and convenience layers for SoX.